jackhodgkiss/kolla
# origin/stackhpc/wallaby workflows

- releasenotes: we move to Debian bullseye
- Update master for stable/wallaby
- Add Python3 xena unit tests
- docs: deprecate ppc64le in Wallaby
- Disable trove for Debian/binary
- Modify healthcheck_mariadb
- Pin erlang
- move to Debian 'bullseye'
- Install Monasca datasource plugin for Grafana
- neutron: Remove vmware-nsx and networking-mlx from binary
- Revert "deprecate use of GnuPG ids for APT keys"
- CI: Remove kolla_python_version from Debian/Ubuntu jobs
- [CI] Fix Ubuntu cephadm jobs names
- CentOS on AArch64 has only Erlang 22
- Fix RabbitMQ and Erlang pins again
- docs: update supported OS distros
- Install RabbitMQ repo key on Ubuntu and Debian
- Revert "CI: temporarily disable CentOS/AArch64 testing"
- CI: move to Debian 'bullseye' nodes
- CI: drop bashism from template overrides
- [CI] Trust only infra mirrors
- Revert "CI: drop ubuntu-aarch64 job"
- setup.cfg: Replace dashes with underscores
- Deprecate tempest and rally
- base: check for compatible distro using just sh
- [release] Use UCA Wallaby
- centos: do not install CentOS Rabbitmq repo twice on AArch64
- gnocchi-base: numpy provides aarch64 binary wheels
- Drop ppc64le support
- gnocchi: add python3-rados to gnocchi-base source image
- add openstack-python3-xena-jobs-arm64 job
- centos: clean up repo enable/disable/keys
- Drop monasca-grafana image.
- [Security] Fix open redirect in Prometheus
- Revert "CI: add monasca-grafana to allowed-to-fail images"
- ubuntu: unpin OVS
- libvirt: pin edk2-ovmf
- Sort the package lists in openstack-base alphabetically
- docs: Update Freenode to OFTC
- Add missing default policy files for ubuntu-binary-horizon
- Horizon: Install required policy files for heat-dashboard
- Horizon: Install policy files for manila-ui
- Fix _get_images_dir when pip-installed with --user
- [CI] Add grafana to allowed-to-fail
- Enable Debian updates repo
- Change network_mode default to host
- Revert "[CI] Add grafana to allowed-to-fail"
- horizon: Correct location of monitoring_policy file
- Remove panko
- [docs] Fix Debian release name
- Pin td-agent to 4.0.* to fix missing logs
- docs: Update release management for Xena process
- Improve offline build scenario.
- Revert "libvirt: pin edk2-ovmf"
- Remove rally
- tox: Use allowlist_externals instead of whitelist_externals
- Bump up cAdvisor's version
- Remove tempest image
- setDaemon() is deprecated in Python 3.10
- [CI] Do not set ansible_python_interpreter for Zuul
- Clean up extra spaces
- Fix build of debian images
- Remove last mongo refs
- CI: Fix epel url override
- Switch OPENSTACK_RELEASE back to master
- Add missing default policy files for debian-binary-horizon
- Add masakari-dashboard to debian binary horizon image
- Fix prometheus-elasticsearch-exporter container.
- Fix missing templating block for kolla-toolbox.
- Add boto3 as s3 dependency for Glance container
- drop leftovers of RHEL support
- test-requirements: remove not needed entries
- Update code to Jinja 3.0 style
- Remove patrole from version-check
- Fix missing pacemaker-cli-utils in Debian hacluster images
- Upgrade prometheus elasticsearch exporter to 1.2.1
- Config: allow to set group for user
- base/deb: bump 'system' groups limit beyond Kolla ones
- Add some system users to fix user uid inconsistencies
- Config: Set default home to /var/lib/{user}
- monasca-thresh: Allow topology check and removal in storm
- Remove Prometheus v1 image
- [horizon] Copy Masakari policy in YAML format
- Fix naming of CentOS Stream in reno
- [release] Use UCA Xena
- ironic-neutron-agent: enable for Debuntu binary
- [release] Use RDO master Delorean packages
- Allow build debian/ubuntu-binary cloudkitty images
- Always use the distro-provided libvirt-python
- Add example of switching CentOS binary images to current release
- Remove chrony image from kolla
- Debian/Ubuntu: handle APT keys in proper way
- Revert "Update code to Jinja 3.0 style"
- docs: add feature freeze to release management
- Revert "Config: Set default home to /var/lib/{user}"
- haproxy: Use haproxy 2.2.x
- Install UCA GPG key
- Fix Elasticsearch Curator
- Move clustercheck script to mariadb-base
- [doc] Ubuntu has Ceph Pacific in Wallaby+
- ceph: Update CentOS packages to Pacific
- Pin td-agent to 4.0.* also on Debian
- debian: Change influxdb, rabbitmq and td-agent repos to bullseye
- toolbox: Move custom Ansible config to global location
- [CI] Drop support for publishing versioned image tags
- toolbox: Move to ansible-core 2.11 and collections
- Do not use distro_python_version for interpreter
- Add proxysql image
- Fix stale commented-out repo entry
- [debian] Use upstream MariaDB
- Debian: use Xena binary packages
- Avoid the use of --force in groupadd
- Move all distros to upstream MariaDB 10.5
- Build source type images by default
- logstash: pin to 7.9.* for Elasticsearch OSS compatibility
- [placement] Stop copying unused script
- [bifrost] Drop workarounds
- Remove neutron api-paste.ini from old location
- release: Use Xena sources by default
- Use RDO's Xena Delorean packages
- Fix GitHub tarball URL for monasca-thresh
- Tidy up renos before Xena release
- Add missing reno
- gnocchi: update to 4.4.1
- Revert "Add missing reno"
- feat: automatic update of workflows
- feat: automatic update of community files
- feat: automatic update of workflows
- feat: automatic update of workflows
